### PR TITLE
Exception handling and minor fixes

### PR DESCRIPTION
**Pull request by trebidav**

### CODE REVIEW

File: developers_chamber/ecs_utils.py [hunk 1]
```diff
@@ -93,23 +94,24 @@ def register_new_task_definition(task_definition, image, region):
...
-            'More than one service with the same name found: {}'.format(
-                len(services['services']),
-            )
+            'More than one service with the same name found: {}'.format(len(services['services']))
         )
     elif number_of_services == 0:
-        raise ClickException(
-            'No services with the name \'{}\' found.'.format(service)
-        )
+        raise ClickException('Service not found: \'{}\''.format(service))
```
Comment (matllubos) [inline developers_chamber/ecs_utils.py]: Service not found: "{}"

File: developers_chamber/ecs_utils.py [hunk 2]
```diff
@@ -144,6 +150,10 @@ def update_service_to_new_task_definition(cluster, service, task_definition, reg
             taskDefinition=task_definition,
             forceNewDeployment=True,
         )
+    except ecs_client.exceptions.ClusterNotFoundException:
+        raise ClickException('Cluster not found: \'{}\''.format(cluster))
```
Comment (matllubos) [inline developers_chamber/ecs_utils.py]: here too

File: developers_chamber/ecs_utils.py [hunk 3]
```diff
@@ -437,18 +478,58 @@ def run_task_and_wait_for_success(cluster, task_definition, command, name, succe
...
+        service_tasks = get_tasks_for_service(cluster=cluster, service=service, region=region)
+        if not service_tasks:
+            LOGGER.info('No active tasks found in service \'{}\''.format(service))
+            continue
+        tasks += service_tasks
+        stop_service(cluster=cluster, service=service, region=region)
+
+    threads = []
+
+    for task in tasks:
```
Comment (matllubos) [inline developers_chamber/ecs_utils.py]: will be better to use thread pool https://www.codementor.io/lance/simple-parallelism-in-python-du107klle

Comment (trebidav) [reply developers_chamber/ecs_utils.py]: what is the problem with threading? As the AWS API calls are I/O blocking, it's better to run more threads than the number of available cores. BTW this PR is WIP - the actual PR adding threads is here: https://github.com/druids/developers-chamber/pull/32

Comment (rubickcz) [reply developers_chamber/ecs_utils.py]: Have you considered asyncio?

Comment (trebidav) [reply developers_chamber/ecs_utils.py]: yes and it's overkill for this easy task

Comment (trebidav) [reply developers_chamber/ecs_utils.py]: Also I am not sure that async ECS Client is supported by boto3

Comment (trebidav) [reply developers_chamber/ecs_utils.py]: fixed in #32

**Pull request merged**